implementations/elixir/ockam/ockam/lib/ockam/session/spawner.ex

```diff
@@ -13,14 +13,14 @@ defmodule Ockam.Session.Spawner do
     @spec spawner_setup(options :: Keyword.t(), state :: map) :: {:ok, options :: Keyword.t(), state :: map()} | {:error, reason :: any()}
     defaults to `&default_spawner_setup`
 
-  Upon receiving a message, `worker_mod` worker will be started
+  Upon receiving a message, `data_worker_mod` worker will be started
```

Comment:
Spawner does not have a handshake and data stages, so this should be just `worker_mod`

Reply:
I think it's fixed now 🤔